app/models/setting/rh_cloud.rb

```diff
@@ -4,8 +4,11 @@ class Setting::RhCloud < Setting
   def self.default_settings
     return unless ActiveRecord::Base.connection.table_exists?('settings')
     return unless super
```

Comment:
BTW this has been moved to `load_defaults`, this is useless now as super here is always `[]`